virtual_alumni.html

```diff
@@ -0,0 +1,18 @@
+---
+layout: default
+group: "navigation"
+title: Virtual Alumni
+permalink: /lunch
```

Comment:
Change this to some other url. The alumni already used `/lunch`, so they would conflict.